inference-engine/tests/ie_test_utils/common_test_utils/ngraph_test_utils.cpp

```diff
@@ -0,0 +1,242 @@
+// Copyright (C) 2018-2021 Intel Corporation
```

Comment:
### Please do not review this file!
This is a temporary solution for accuracy check inside transformations tests. It is required by complicated trasnformations like ric fusion. But this code will be aligned with common utils but in other PR.